Description: Excludes the groovyx.gpars.remote.netty classes which rely on an
 unsupported version of Netty (these classes are never used in Debian anyway).
 The next upstream release will support a more recent version of Netty.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -48,6 +48,15 @@
 sourceCompatibility = 6
 targetCompatibility = 6
 
+sourceSets {
+    main {
+        groovy {
+            srcDir 'src/main/groovy'
+            exclude '**/remote/netty/**'
+       }
+   }
+}
+
 def theVendor = 'gpars.org'
 def theTitle = 'GPars: Groovy Parallel Systems'
 
